com.github.dariasteam.cowsrevenge.json

```diff
@@ -0,0 +1,142 @@
+            builtin_libvpx=no \\
+            builtin_zlib=no \\
+            builtin_opus=yes \\
+            builtin_speex=yes \\
+            builtin_squish=yes \\
+            pulseaudio=yes \\
+            udev=no \\
+            tools=yes",
+            "install -D -m755 bin/godot* /app/bin/godot-bin"
+      ]
+    },
+    {
+      "name": "godot-release-template",
+      "buildsystem": "simple",
+      "sources": [
+          {
+          "type": "git",
+          "url": "https://github.com/godotengine/godot",
+          "commit": "a524cc92e7cccd012d028d870e4648c59473d04e",
+          "branch": "2.1"
+        }
+      ],
+      "build-commands": [
+          "scons platform=x11 \\
+            CCFLAGS=-I/app/include \\
+            prefix=/app \\
+            unix_global_settings_path=/app \\
+            verbose=yes \\
+            -j `nproc` \\
+            target=release \\
+            use_llvm=yes \\
+            builtin_freetype=no \\
+            builtin_libmpcdec=yes \\
+            builtin_libogg=no \\
+            builtin_libpng=no \\
+            builtin_libtheora=no \\
+            builtin_libvorbis=no \\
+            builtin_libwebp=no \\
+            builtin_libvpx=no \\
+            builtin_zlib=no \\
+            builtin_opus=yes \\
+            builtin_speex=yes \\
+            builtin_squish=yes \\
+            pulseaudio=yes \\
+            udev=no \\
+            tools=no",
+          "install -D -m755 bin/godot.x11.opt.*.llvm /app/templates/linux_x11_`getconf LONG_BIT`_release"
+      ]
+    },
+    {
+      "name": "cowsrevenge",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/Dariasteam/Cows-Revenge",
+          "commit" : "dc4e65763653d0fff72379532685a8c2391ca05b"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/cowsrevenge",
+        "mv * /app/cowsrevenge/",
+        "cd /app/cowsrevenge/; \\
+          ../bin/godot-bin -export 'Linux X11' cowsrevenge",
+
+        "install -D -m755 /app/cowsrevenge/cowsrevenge /app/bin/",
+        "install -D -m644 /app/cowsrevenge/linux_build/com.github.dariasteam.cowsrevenge.desktop \\
+                          /app/share/applications/com.github.dariasteam.cowsrevenge.desktop",
+        "install -D -m644 /app/cowsrevenge/linux_build/com.github.dariasteam.cowsrevenge.appdata.xml \\
+                          /app/share/metainfo/com.github.dariasteam.cowsrevenge.appdata.xml",
+        "install -D -m755 /app/cowsrevenge/linux_build/cowsrevenge_icon_64.png \\
+                          /app/share/icons/hicolor/64x64/apps/com.github.dariasteam.cowsrevenge.png",
+        "install -D -m755 /app/cowsrevenge/linux_build/cowsrevenge_icon_128.png \\
+                          /app/share/icons/hicolor/128x128/apps/com.github.dariasteam.cowsrevenge.png",
+
+        "rm -rf /app/cowsrevenge",
+        "rm -rf /app/bin/godot-bin",
+        "rm -rf /app/templates"
```

Comment:
I think these better belong in the `cleanup` property of godot and its templates.